mail-client/alpine/alpine-2.00-r7.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit flag-o-matic autotools toolchain-funcs
+
+CHAPPA_PL=115
+DESCRIPTION="alpine is an easy to use text-based based mail and news client"
+HOMEPAGE="http://www.washington.edu/alpine/ http://patches.freeiz.com/alpine/"
+SRC_URI="ftp://ftp.cac.washington.edu/alpine/${P}.tar.bz2
+	chappa? ( http://patches.freeiz.com/alpine/patches/alpine-2.00/all.patch.gz
+	-> ${P}-chappa-${CHAPPA_PL}-all.patch.gz )"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc ipv6 kerberos ldap libressl nls onlyalpine pam passfile smime spell ssl threads topal +chappa"
+
+DEPEND="pam? ( virtual/pam )
+	>=net-libs/c-client-2007f-r4[topal=,chappa=]
+	>=sys-libs/ncurses-5.1:0=
+	ssl? (
+		!libressl? ( dev-libs/openssl:0 )
+		libressl? ( dev-libs/libressl )
+	)
+	ldap? ( net-nds/openldap )
+	kerberos? ( app-crypt/mit-krb5 )
+	spell? ( app-text/aspell )
+	topal? ( >=net-mail/topal-72 )"
+RDEPEND="${DEPEND}
+	app-misc/mime-types
+	!onlyalpine? ( !mail-client/pine )
+	!<=net-mail/uw-imap-2004g"
+
+pkg_setup() {
+	if use smime && use topal ; then
+		ewarn "You can not have USE='smime topal'. Assuming topal is more important."
+	fi
+}
+
+src_prepare() {
+	use chappa && eapply "${WORKDIR}/${P}-chappa-${CHAPPA_PL}-all.patch"
+	use topal && eapply /usr/share/topal/patches/"${P}".patch-{1,2}
+
+	# do not use the bundled c-client
+	ebegin "Unbundling the c-client library"
+	rm -rf "${S}"/imap
+	local f
+	while read f ; do
+	sed -i -e \
+		's~^#include[[:blank:]]".*/c-client/\(.*\)"~#include <imap/\1>~g' "$f"
+	done < <(find "${S}" -name "*.c" -o -name "*.h")
+	eend $?
+
+	! use pam || eapply "${FILESDIR}"/2.00-lpam.patch
```

Comment:
@vaeth hey. Could you please do something a bit wordier yet easier to read for all?
```if ! use pam; then
  epatch ...
fi```